compiler/x/env/OMRCPU.cpp

```diff
@@ -126,3 +150,57 @@ OMR::X86::CPU::supportsTransactionalMemoryInstructions()
    flags32_t processorFeatureFlags8(self()->getX86ProcessorFeatureFlags8());
    return processorFeatureFlags8.testAny(TR_RTM);
    }
+
+bool
+OMR::X86::CPU::isAMDK6()
+   {
+   return self()->is(OMR_PROCESSOR_X86_AMDK6) || self()->is(OMR_PROCESSOR_X86_AMDK5);
+   }
```

Comment:
This seems a little odd. `isAMDK6()` answers true if the process is `AMDK5`?

Reply:
The code in OpenJ9 that I referenced:
```cpp
  bool isAMDK6()             { return (_processorDescription & 0x000000fe) == TR_ProcessorAMDK5; } // accept either K5 or K6
```
- However it's not used anywhere, I guess I don't even need to include this API..

Reply:
`isAMDK6()` removed.